src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -189,9 +222,8 @@ public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesD
             new object[] { Configuration.Http.WrongHostNameCertRemoteServer },
         };
 
-        [ActiveIssue(7812, TestPlatforms.Windows)]
         [OuterLoop] // TODO: Issue #11345
-        [Theory]
+        [ConditionalTheory(nameof(ClientSupportsDHECipherSuites))]
         [MemberData(nameof(CertificateValidationServers))]
         public async Task NoCallback_BadCertificate_ThrowsException(string url)
```

Comment:
Disabling this test seems weird.  If no cipher suite was negotiated, wouldn't an exception be thrown? So wouldn't this test just work?